src/widget/form/addfriendform.h

```diff
@@ -55,9 +55,9 @@ class AddFriendForm : public QObject
     bool addFriendRequest(const QString& friendAddress, const QString& message);
 
 signals:
-    void friendRequested(const QString& friendAddress, const QString& message);
-    void friendRequestAccepted(const QString& friendAddress);
-    void friendRequestsSeen();
+    void friendRequested(const QString& friendAddress, const QString& message) const;
+    void friendRequestAccepted(const QString& friendAddress) const;
+    void friendRequestsSeen() const;
```

Comment:
Basically yes. Btw. found a nice article here: https://isocpp.org/wiki/faq/const-correctness#const-cast-and-optimization